packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2015,28 +1982,27 @@ export class ContainerRuntime
 	 * Parse an op's type and actual content from given serialized content
 	 * ! Note: this format needs to be in-line with what is set in the "ContainerRuntime.submit(...)" method
 	 */
-	private parseOpContent(serializedContent?: string): ContainerRuntimeMessage {
-		assert(serializedContent !== undefined, 0x6d5 /* content must be defined */);
-		const { type, contents, compatDetails }: ContainerRuntimeMessage =
-			JSON.parse(serializedContent);
-		assert(type !== undefined, 0x6d6 /* incorrect op content format */);
-		return { type, contents, compatDetails };
+	private parseLocalOpContent(serializedContents?: string): LocalContainerRuntimeMessage {
+		assert(serializedContents !== undefined, 0x6d5 /* content must be defined */);
+		const message: LocalContainerRuntimeMessage = JSON.parse(serializedContents);
```

Comment:
This could be a behavior change if the object has more props than those 3.

Reply:
But it's "probably" fine based on my memory of callers, who pull the 3 pieces off it, and don't pass the object itself around.

Reply:
Per some discussion this probably better. If properties are added they would be dropped with prior logic.
The updated types make it hard to get full safety when handling properties one-by-one like before which is exactly true apart from just going with what you've got as we now do.